lr_face/data_providers.py

```diff
@@ -0,0 +1,146 @@
+import csv
+from dataclasses import dataclass
+from typing import Tuple
+import os
+
+import pandas as pd
+import cv2
+import numpy as np
+from sklearn.model_selection import GroupShuffleSplit
+
+
+@dataclass
+class Images:
+    y_train: np.ndarray
+    X_train: np.ndarray
+    y_calibrate: np.ndarray
+    X_calibrate: np.ndarray
+    y_test: np.ndarray
+    X_test: np.ndarray
+
+
+def test_data(resolution):
```

Comment:
Ik zou de test/dummy op een hardcoded resolutie maken, omdat bij de modellen nu de afbeeldingen gescaled worden naar het juiste formaat voor het model.

Reply:
dan is het toch nog steeds fijn als de test_data een bepaalde resolutie kan zijn? ik zal een default erin zetten, mssn bedoel je dat?